### DISCUSSION TITLE
Loading the Contact Form 7 Signature Addon Plugin Only on certain WordPress pages

### DISCUSSION THREAD

**Discussion by currenge** [szimek/signature_pad#712] [General]
Hi,

We are happily using the  Contact Form 7 Signature Addon Plugin. This is done on 3 pages of our website.

I noticed it seems to upload on all website pages.

I would like this plugin to load, or possibly even Activate only on three certain pages, which I may identify, for example according to a fetching as below:

```
global $post;

if ($post->post_type == 'page'){
	if ($post->ID == "2808") {
               include ...
```

And, at the same time possibly deactivate or have the plugin, not upload as usual, where that would have mean uploading on all website pages.

I am now reading a possible general explanation on:

https://kinsta.com/blog/disable-wordpress-plugins-loading/

https://developer.wordpress.org/plugins/hooks/filters/

May I ask, how could this goal be achieved?

**Comment by UziTech** [comment]
I think you have the wrong repo. I've never heard of Contact Form 7.

**Reply by currenge** [reply]
Hi Uzi,

My apologies. You are right.
